mmdet3d/models/losses/iou_loss_axis_aligned.py

```diff
@@ -0,0 +1,109 @@
+import torch
+from torch import nn as nn
+
+from mmdet.models.builder import LOSSES
+
+
+def iou_loss_axis_aligned(center_preds,
+                          size_preds,
+                          center_targets,
+                          size_targets,
+                          weight=1.0,
+                          reduction='mean'):
+    """Calculate the IoU loss (1-IoU) of two axis aligned bounding boxes.
+
+    Args:
+        center_preds (torch.Tensor): Center predictions with shape [B, N, 3].
+        size_preds (torch.Tensor): Size predictions with shape [B, N, 3].
+        center_targets (torch.Tensor): Center targets (gt) \
+            with shape [B, N, 3].
+        size_targets (torch.Tensor): Size targets with shape [B, N, 3].
+        weight (torch.Tensor or float): Weight of loss.
```

Comment:
weight (torch.Tensor|float, optional): Weight of loss. Defaults to 1.0.
reduction (str, optional): xxxxx. Defaults to 'mean'.